package.json

```diff
@@ -8,7 +8,7 @@
   "scripts": {
     "dev": "gulp dev --gulpfile 'app/gulpfile.js'",
     "heroku": "node app/start.js",
-    "test": "standard && gulp compile && jest",
+    "test": "standard & gulp compile && jest",
```

Comment:
Is this a shell script? If so, what happens if `standard` fails? It looks as if failures might be missed...